applications/mne_analyze/plugins/coregistration/coregistration.cpp

```diff
@@ -0,0 +1,756 @@
+//=============================================================================================================
+/**
+ * @file     coregistration.cpp
+ * @author   Ruben Dörfel <doerfelruben@aol.com>
+ * @since    0.1.6
+ * @date     August, 2020
+ *
+ * @section  LICENSE
+ *
+ * Copyright (C) 2020, Ruben Dörfel. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification, are permitted provided that
+ * the following conditions are met:
+ *     * Redistributions of source code must retain the above copyright notice, this list of conditions and the
+ *       following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+ *       the following disclaimer in the documentation and/or other materials provided with the distribution.
+ *     * Neither the name of MNE-CPP authors nor the names of its contributors may be used
+ *       to endorse or promote products derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
+ * WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ *
+ * @brief    Definition of the CoRegistration class.
+ *
+ */
+
+//=============================================================================================================
+// INCLUDES
+//=============================================================================================================
+
+#include "coregistration.h"
+
+#include <anShared/Management/analyzedata.h>
+#include <anShared/Management/communicator.h>
+#include <anShared/Utils/metatypes.h>
+#include <anShared/Model/bemdatamodel.h>
+
+#include "disp/viewers/coregsettingsview.h"
```

Comment:
External usage of a library (from an application or other library) such as `#include "disp/viewers/coregsettingsview.h"` should always use <>. For example: `#include <disp/viewers/coregsettingsview.h>. Please check rest of PR as well.